File: 392-is-subsequence/is-subsequence.cs
public class Solution {
    public bool IsSubsequence(string s, string t) {
        int i = 0;// index for s string ;
        int j = 0;// index for t string
        while(i<s.Length && j<t.Length)
        {
            if(s[i]==t[j])
            {
                i++;
            }
            j++;

        }
        return i==s.Length;
  
        
    }
}

File: 392-is-subsequence/is-subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
         int i = 0 ;//index for s string;
         int j = 0 ;//index for t string
         while(i<s.length() && j < t.length())
         {
            if(s.charAt(i)==t.charAt(j))
            {
                i++;

            }
            j++;
         }
         return i==s.length();

    }
}